### PR TITLE
Add IP address (network) label to XRootD access metrics

### PR DESCRIPTION
**Pull request by patrickbrophy**
This PR addresses #1425. This PR adds a IP address (netwrok) label to XRD access metrics (xrootd_transfer_readv_segments_count, xrootd_transfer_operations_count, and xrootd_transfer_bytes). This PR also updated some tests to handle the new label. This PR also addresses comments from @bbockelm in PR #1445.

### CODE REVIEW

File: metrics/xrootd_metrics.go
Comment (haoming29) [inline metrics/xrootd_metrics.go:285]: Any reason we want to name it `network` instead of something like `ip`?

Comment (patrickbrophy) [reply metrics/xrootd_metrics.go:285]: @bbockelm Had mentioned using the `network` label in #1445, 
> given we are using networks and not hostnames, let's rename the labels to "network" perhaps?  Looking around on Google, it's not clear if there's a standard practice here.

**Review by haoming29 — APPROVED** [review]
LGMT. Let's wait until #1469 is merged in and OSX tests are passing to merge this in

**Comment by haoming29** [comment]
@patrickbrophy I think your rebase looks a bit off. You are catching a bunch of previous commits.

**Pull request merged**
